src/metabase/util/malli.clj

```diff
@@ -87,16 +88,24 @@ explain-fn-fail!
   [:and any?
    [:fn {:description "a malli schema"} mc/schema]])
 
-(defn with-api-error-message :- Schema
+(def ^:private localized-string-schema
+  [:fn {:error/message "must be a localized string"}
+   i18n/localized-string?])
+
+(defn with-api-error-message
   "Update a malli schema to have a :description (picked up by api docs),
-  and a :error/message (used by defendpoint). They don't have to be the same, but usually are."
-  ([mschema :- Schema message :- string?]
-   (with-api-error-message mschema message message))
-  ([mschema :- Schema
-    docs-message :- string?
-    error-message :- string?]
-   (mut/update-properties mschema assoc
-                          ;; override generic description in api docs
-                          :description docs-message
-                          ;; override generic description in defendpoint api errors
-                          :error/message error-message)))
+  and a :error/message (used by defendpoint). They don't have to be the same, but usually are.
```

Comment:
Maybe a nit, but this is out of date now that we are associng the `:error/fn` property, and not the `:error/message` property.